dune/xt/functions/lambda/global-flux-function.hh

```diff
@@ -0,0 +1,145 @@
+// This file is part of the dune-xt-functions project:
+//   https://github.com/dune-community/dune-xt-functions
+// Copyright 2009-2017 dune-xt-functions developers and contributors. All rights reserved.
+// License: Dual licensed as BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+//      or  GPL-2.0+ (http://opensource.org/licenses/gpl-license)
+//          with "runtime exception" (http://www.dune-project.org/license.html)
+// Authors:
+//   Felix Schindler (2013 - 2017)
+//   Rene Milk       (2013 - 2016)
+//   Sven Kaulmann   (2013)
+//   Tobias Leibner  (2014)
+
+#ifndef DUNE_XT_FUNCTIONS_LAMBDA_GLOBAL_FLUX_FUNCTION_HH
+#define DUNE_XT_FUNCTIONS_LAMBDA_GLOBAL_FLUX_FUNCTION_HH
+
+#include <functional>
+
+#include <dune/xt/common/memory.hh>
+
+#include <dune/xt/functions/interfaces.hh>
+
+namespace Dune {
+namespace XT {
+namespace Functions {
+
+
+template <class U, size_t state_derivative_order = 0, class R = typename U::RangeFieldType, size_t r = 1, size_t rC = 1>
+class GlobalLambdaFluxFunction
+{
+  static_assert(AlwaysFalse<U>::value, "Not available for these dimensions or this state_derivative_order!");
+};
+
+/**
+ * \brief A flux function given by a lambda expression which can be evaluated without an entity.
+ *
+ *        To model the flux F in a Burgers equation, F(u) := u^p, with a variable exponent p given a localizable
+ *        function u, use as
+ * \code
+typedef some_localizable_function U;
+typedef LocalLambdaFluxFunction<U> FluxType;
```

Comment:
this should read `GlobalLambdaFluxFunction` instead of `LocalLambdaFluxFunction`